src/components/molecules/OcBreadcrumb/OcBreadcrumb.spec.js

```diff
@@ -20,15 +20,49 @@ describe("OcBreadcrumb", () => {
     expect(wrapper.props().variation).toMatch("lead")
     expect(wrapper.classes()).toContain("oc-breadcrumb-lead")
     expect(wrapper).toMatchSnapshot()
-  }),
-    it("displays all items", () => {
-      const wrapper = shallowMount(Breadcrumb, {
-        propsData: {
-          items,
-        },
-      })
+  })
+  it("displays all items", () => {
+    const wrapper = shallowMount(Breadcrumb, {
+      propsData: {
+        items,
+      },
+    })
+
+    expect(wrapper.findAll(".oc-breadcrumb-list-item").length).toBe(4)
```

Comment:
The static check for `4` elements makes this test quite brittle. Can you check with `items.length`?